src/Plugin.php

```diff
@@ -53,6 +53,8 @@ public static function init() {
     add_action('switch_theme', __CLASS__ . '::purgeAll');
 
     add_action('shutdown', __CLASS__ . '::executePurge');
+
+    add_filter('gform_after_save_form', __CLASS__ . '::gform_after_save_form', 10, 2);
```

Comment:
```suggestion
    // Invalidate all pages containing a gravityform upon saving.
    add_filter('gform_after_save_form', __CLASS__ . '::gform_after_save_form', 10, 2);
```

Reply:
This should also be moved further up - as usual, the actions are registered in their natural execution order.